ios/GLCanvas.m

```diff
@@ -320,7 +321,9 @@ - (void)drawRect:(CGRect)rect
     });
   }
   else {
+    RCTProfileBeginEvent(0, @"GLCanvas render", nil);
     [self render];
+    RCTProfileEndEvent(0, @"gl", nil);
```

Comment:
It's category's name (see the screenshot attached). I don't see any pattern or convetion in the RN source code. We could add more events later in the same category.